package.json

```diff
@@ -1,145 +1,25 @@
+    "format": "turbo run format",
+    "test": "turbo run test"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@storybook/addon-a11y": "^6.4.19",
-    "@storybook/addon-essentials": "~6.4.19",
-    "@storybook/addon-storysource": "~6.4.19",
-    "@storybook/addons": "~6.4.19",
-    "@storybook/api": "~6.4.19",
-    "@storybook/builder-webpack5": "~6.4.19",
-    "@storybook/components": "~6.4.19",
-    "@storybook/core-events": "~6.4.19",
-    "@storybook/manager-webpack5": "~6.4.19",
-    "@storybook/preset-typescript": "^3.0.0",
-    "@storybook/react": "~6.4.19",
-    "@storybook/theming": "~6.4.19",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.0.0",
-    "@testing-library/react-hooks": "^7.0.1",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/classnames": "^2.2.7",
-    "@types/jest": "^27.0.0",
-    "@types/node": "^10.12.2",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.3",
-    "@types/react-transition-group": "^2.9.2",
-    "@typescript-eslint/eslint-plugin": "^4.31.1",
-    "@typescript-eslint/parser": "^4.31.1",
-    "babel-loader": "^8.2.2",
-    "concurrently": "^5.3.0",
-    "cpx": "^1.5.0",
-    "creevey": "^0.8.0-beta.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-react": "^7.25.1",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-storybook": "^0.5.7",
-    "fast-glob": "^3.2.5",
-    "husky": "^4.0.0",
-    "inquirer": "^6.2.2",
-    "jest": "^27.0.6",
-    "jest-cli": "^27.0.6",
-    "jest-junit": "^12.0.0",
-    "lint-staged": "^12.1.2",
-    "markdown-to-jsx": "6.11.4",
-    "mkdirp": "^1.0.4",
-    "prettier": "2.2.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "rimraf": "^2.6.2",
-    "storybook-dark-mode": "^1.0.9",
-    "tippy.js": "^6.3.1",
-    "ts-jest": "^27.0.4",
-    "ts-loader": "^8.0.1",
-    "ts-node": "^8.0.2",
-    "typescript": "^4.4.3",
+    "turbo": "latest",
     "webpack": "5"
   },
-  "peerDependencies": {
-    "react": "^16.8.6 || ^17.0.0",
-    "react-dom": "^16.8.6 || ^17.0.0"
-  },
-  "resolutions": {
-    "braces": "^2.3.1",
-    "hosted-git-info": ">=3.0.8",
-    "trim": "^0.0.3",
-    "browserslist": "^4.16.6",
-    "glob-parent": "^5.1.2",
-    "css-what": "^5.0.1",
-    "immer": "^9.0.6",
-    "set-value": "^4.1.0",
-    "prismjs": "^1.25.0",
-    "nth-check": "^2.0.1",
-    "ansi-regex": "^5.0.1",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.1",
-    "postcss": "8",
-    "nanoid": "^3.1.31"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{tsx,ts,jsx,js}": [
-      "prettier --write",
-      "eslint --fix",
-      "yarn lint:copyright --fix"
-    ]
+  "engines": {
+    "npm": ">=7.0.0",
+    "node": ">=16.0.0"
   },
-  "sideEffects": [
-    "**/*.scss",
-    "**/*.css"
-  ]
+  "packageManager": "yarn@1.22.18"
```

Comment:
Have you checked what is a prefered package manager? Maybe we should switch to npm or pnpm.

Reply:
i think pnpm is the preferred package manager, but i kept yarn because that's what we were using before

Reply:
Could you investigate main differences between pnpm and yarn because more and more I am seeing people saying to not 
 to use yarn

Reply:
Maybe not in this PR? I would need to update all the documentation and workflows everywhere.

It's just a package manager so it could easily be done later. There is even a command to migrate lockfile: https://pnpm.io/cli/import